pyaedt/edb_core/EDB_Data.py

```diff
@@ -13,6 +16,431 @@
     )
 
 
+class EDBNetsData(object):
+    """Manages EDB functionalities for a primitives.
+    It Inherits EDB Object properties.
+
+    Examples
+    --------
+    >>> from pyaedt import Edb
+    >>> edb = Edb(myedb, edbversion="2021.2")
+    >>> edb_net = edb.core_nets.nets["GND"]
+    >>> edb_net.name # Class Property
+    >>> edb_net.GetName() # EDB Object Property
+    """
+
+    def __getattr__(self, key):
+        if key in dir(self):
+            try:
+                return getattr(self, key)
+            except:
+                raise AttributeError("Attribute not present")
```

Comment:
When is it going in this `except` considering the `if` at line 33?

Reply:
when you pass a wrong argument